third_party/move/move-compiler-v2/tests/acquires-checker/acquires_error_msg.exp

```diff
@@ -0,0 +1,9 @@
+
+Diagnostics:
+error: missing acquries annotation for Test
+  ┌─ tests/acquires-checker/acquires_error_msg.move:6:16
+  │
+6 │     public fun call_modify_without_acquire() {
+  │                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+7 │         modify(); // expect error message here
+  │         -------- acquired by call
```

Comment:
Might be even better if you can link to the method called.

Reply:
What do we mean by link?